packages/deployments/contracts/contracts/messaging/connectors/SpokeConnector.sol

```diff
@@ -54,6 +54,8 @@ abstract contract SpokeConnector is Connector, ConnectorManager, WatcherClient,
 
   event DelayBlocksUpdated(uint256 indexed updated, address caller);
 
+  event SnapshotRootSaved(uint256 snapshotId, bytes32 root, uint256 count);
```

Comment:
we should index the ids and roots (maybe count but eh)

Reply:
👀